lib/hanrahashiv.py

```diff
@@ -360,13 +358,12 @@ def ReplaceItemsInFormula(
         *new_formula_tex_string_list,
         font_size=formula[0].font_size, color=formula.color, tex_template=formula.tex_template
     )
-    new_formula.move_to(formula).align_to(formula, DL)
+    new_formula.move_to(formula).align_to(formula, alignment)
 
     for i in range(len(formula)):
         new_formula[i].set_color(formula[i].color)
-    if len(colors) > 0:

```

Comment:
Why have you removed the if statement?

Reply:
Ես գրել եմ `for i in range(len(colors)):`, եթե `len(colors)`-ը մեծ չլինի 0-ից մեկա loop-ը չի աշխատի, պատճառը ո՞րնա, դրա վերևում էլ գրել `if len(colors) > 0:`

Reply:
կրեցիր

տակի if֊ն ու իրա suite֊ը տարբեր տողերի վրա կգրե՞ս